ext-apiserver/sample/machineset.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: cluster.k8s.io/v1alpha1
+kind: MachineSet
+metadata:
+  name: machineset-example
+spec:
```

Comment:
should the example actually have a spec defined?

Reply:
Just a placer holder. maybe mrincompetent@ can add actual content here.

Reply:
We could use https://github.com/p0lyn0mial/kube-deploy/blob/08c218162944d5c873fb49b8d758278a7253a47e/cluster-api/examples/manifests/machinesets.yaml